### DISCUSSION TITLE
Umi@2作为qiankun的子应用

### DISCUSSION THREAD

**Discussion by yuki070** [umijs/umi#5988] [Q&A] [answered]
请问一个Umi@2的项目，如果作为qiankun的子应用，应该从哪里把bootstrap、mount、unmount导出？

尝试了在src/app.js中导出

```js
export async function bootstrap() {
  console.log('bright bootstrap...');
}

export async function mount(props) {
  console.log('bright mount...', props);
}

export async function unmount(props) {
  console.log('bright unmount...', props);
}
```

但跑起来会报如下的错误
![image](https://user-images.githubusercontent.com/656768/104542908-5bfb2680-565f-11eb-99b4-5fe9d5245410.png)

**Comment by yuki070** [comment]
umi2里能使用`@umijs/plugin-qiankun`这个插件吗？

**Comment by sorrycc** [comment, marked as answer]
https://github.com/umijs/umi-plugin-qiankun
有对应的 umi 2 版本，参考这里的安装说明。
